pom.xml

```diff
@@ -17,15 +17,15 @@
         <spark.version.minor>1</spark.version.minor>
 
         <!-- other dependencies -->
-        <kusto-sdk.version>3.2.1</kusto-sdk.version>
+        <kusto-sdk.version>4.0.3</kusto-sdk.version>
         <specs2.version>3.6.5</specs2.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
-        <azure-storage.version>8.6.5</azure-storage.version>
+        <azure-storage.version>8.6.6</azure-storage.version>
         <keyvault-secrets.version>4.1.5</keyvault-secrets.version>
         <azure-identity.version>1.2.4</azure-identity.version>
         <commons-lang3.version>3.9</commons-lang3.version>
-        <fasterxml.jackson.version>[2.11.0,2.12.0)</fasterxml.jackson.version>
+        <fasterxml.jackson.version>[2.11.0,2.13.0)</fasterxml.jackson.version>
```

Comment:
Please do check this once with Databricks Jackson for library compatibility.

Reply:
They using 2.13.4 
@ag-ramachandran

Reply:
then lets use 2.13.4 as well